src/ripple/rpc/handlers/Version.h

```diff
@@ -0,0 +1,63 @@
+//------------------------------------------------------------------------------
```

Comment:
I'll rename rpc/Version to rpc/RPCVersion.h.  (The other choice is renaming the version handler to VersionHandler, but we already have so many things marked as "Handler"...)